packages/protocol/migrations/08_stabletoken.ts

```diff
@@ -46,7 +47,7 @@ module.exports = deploymentForCoreContract<StableTokenInstance>(
 
     for (const oracle of config.stableToken.oracles) {
       console.info(`Adding ${oracle} as an Oracle for StableToken`)
-      await sortedOracles.addOracle(stableToken.address, oracle)
+      await sortedOracles.addOracle(stableToken.address, ensureHexLeader(oracle))
```

Comment:
Without the `0x`, the oracle silently doesn't get whitelisted correctly. This needs to be merged before `baklava` deployment.